### DISCUSSION TITLE
Daikin support possible

### DISCUSSION THREAD

**Discussion by kroon040** [arendst/Tasmota#17890] [Ideas]
I'm using a Daikin airco but they are connecting to the cloud. 

Ther is now a solution for the esp32 , so now it should be nice if it can be supported in tasmota?

https://github.com/revk/ESP32-Daikin

**Comment by sfromis** [comment]
Well, what can be supported in Tasmota depends on who wants to do the work on creating a driver. It is very normal that drivers are created by someone with the device, wanting to get it supported.
